FirebaseAuth/Tests/Unit/FIROAuthProviderTests.m

```diff
@@ -14,12 +14,15 @@
  * limitations under the License.
  */
 
-#import <OCMock/OCMock.h>
+#include <TargetConditionals.h>
+#if TARGET_OS_IOS
+
 #import <XCTest/XCTest.h>
 #import "FirebaseAuth/Sources/Public/FIRAuthErrors.h"
 #import "FirebaseAuth/Sources/Public/FIRAuthUIDelegate.h"
 #import "FirebaseAuth/Sources/Public/FIROAuthProvider.h"
 #import "FirebaseCore/Sources/Private/FirebaseCoreInternal.h"
+#import "OCMock.h"
```

Comment:
nit: I remember I received comments about the header imports order to comply with go/objc-style#order-of-includes. It seems  like `#import "OCMock.h"` and `#import <XCTest/XCTest.h>` should be in a separate group from Auth headers.

Reply:
Done.